cmds/core/date/date_test.go

```diff
@@ -173,7 +173,7 @@ func TestRun(t *testing.T) {
 			arg:     []string{"+%W"},
 			univ:    true,
 			fileref: "",
-			expExp:  "\\d\\d",
+			expExp:  "\\d*",
```

Comment:
Can you add some explanation to the commit message on why these are necessary? I assume it's because we started a new year, so week numbers are shorter.